server/src/main/java/com/cloud/api/dispatch/ParamProcessWorker.java

```diff
@@ -70,6 +71,7 @@
     private static final Logger s_logger = Logger.getLogger(ParamProcessWorker.class.getName());
     public final DateFormat inputFormat = new SimpleDateFormat("yyyy-MM-dd");
     public final DateFormat newInputFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+    public static final List<String> avoidList = new ArrayList<>(Arrays.asList("username", "password"));
```

Comment:
How about using ApiConstants.USERNAME and PASSWORD ?

Reply:
Done - Thanks @ravening